test/e2e/components/vl-action-group.js

```diff
@@ -1,10 +1,14 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('selenium-webdriver');
 
 class VlActionGroup extends VlElement {  
     constructor(driver, selector) {
         super(driver, selector);
     }
 
+    async klikOpButtonMetIndex(index) {
+        return (await this.findElements(By.css('button')))[index].click();
+    }
```

Comment:
Ik zou eerder een getButtonMetIndex voorzien die een VlButton page object geeft, zodat je al de functies voorzien bij VlButton kan gebruiken. In het kader van de vl-ui-action-group kan je niet echt een functionele waardevolle functie voorzien voor het klikken op de knoppen, dus zou ik dat niet doen.